spire/templates/apps/dovetail-marketing-form-handler.yml

```diff
@@ -107,6 +107,9 @@ Resources:
                 const email = params.get('inputEmail');
                 const name = params.get('inputName') || 'Unknown';
                 const organization = params.get('textOrganization') || 'Unknown';
+                const type = params.get('messageType') || 'Unknown';
+                const sourceIp = event.requestContext.http.sourceIp;
+                const userAgent = event.requestContext.http.userAgent;
```

Comment:
We get the IP from the form, but I want to have and use the IP from the actual request, and the user agent as well, for reviewing the request.